app/app.js

```diff
@@ -9,6 +10,25 @@ app.get('/api/categories', getCategories)
 
 app.get('/api', getEndpoints)
 
+app.get('/api/reviews/:review_id', getReviewById)
+
+app.use((err, req, res, next) => {
+    if(err.code === '22P02'){
+        res.status(400).send({msg : 'Bad Request!'})
+    }else {
+        next(err)
+    }
+})
+app.use((err,req,res,next)=>{
+    if(err.status && err.msg){
```

Comment:
The very last app.use for errors will be for a 500 error (server error) I would therefore make the status and message more specific for a 500. I would also look to eventually move these out to another file (error handlers) The error handlers should be split by functionality (eg handleCustomErrors, handlePsqlErrors) - but this is up to you!

Reply:
Hey Hannah, Not sure if you noticed, but if this doesn't have a custom error - it will then invoke the next(err) then carry out my server error middleware?

Reply:
You may choose to put these in an error handler file and import it in :)